src/minimega/vm.go

```diff
@@ -512,3 +512,50 @@ func ParseVmState(s string) (VmState, error) {
 
 	return VM_ERROR, fmt.Errorf("invalid state: %v", s)
 }
+
+// Get the VM info from all hosts optionally applying column/row filters.
+// Returns a map with keys for the hostnames and values as the tabular data
+// from the host.
+func globalVmInfo(masks []string, filters []string) map[string][][]string {
```

Comment:
We may still want to keep this helper as it makes it pretty simple to pull info from all hosts. With vm info packing vms.vms into the Data element, this may change to return map[string]vms.